### ISSUE TITLE
Suggestion to use images from any CDN

### ISSUE THREAD

**Issue by ivan-nginx** [theme-next/theme-next.org#28]
As we use Netlify, maybe for saving traffic we must use images / pictures from CDN?

**Comment by stevenjoezhang** [comment]
Same problem here: (https://github.com/theme-next/hexo-theme-next/pull/204)
![2019-03-08 1 07 51](https://user-images.githubusercontent.com/16272760/54008866-6e72c200-41a3-11e9-9fad-54e072072163.png)


Maybe we can use (user-images.githubusercontent.com) , or (github and jsdelivr).

**Comment by ivan-nginx** [comment]
Yes. Need to think about this. Images must be loaded as easy as possible. Then just pasting them into the post. Take a mind, soon will be multilanguage function and possible we will use multilanguage images too.

**Comment by stevenjoezhang** [comment]
And maybe these third-party libraries can also be loaded via jsDeliver CDN?

![屏幕快照 2019-03-23 上午9 52 28](https://user-images.githubusercontent.com/16272760/54860183-89236a00-4d51-11e9-9b46-4b719c19a1d9.png)

**Comment by ivan-nginx** [comment]
Yes! They can. But need strongly version testing for this.

**Comment by jiangtj** [comment]
I find netlify have open-source-policy.
https://www.netlify.com/legal/open-source-policy/

<blockquote><img src="https://www.netlify.com/img/global/meta-image.jpg" width="48" align="right"><div><img src="/img/global/favicon/favicon-16x16.png" height="14"> Netlify</div><div><strong><a href="https://www.netlify.com/legal/open-source-policy/">
  
    Open Source Plan Policy | Netlify
  
</a></strong></div><div>
  
    Deploy modern static websites with Netlify. Get CDN, Continuous deployment, 1-click HTTPS, and all the services you need. Get started for free.
  
</div></blockquote>
